CodeGen/docker_compose/amd/gpu/rocm/README.md

```diff
@@ -0,0 +1,128 @@
+###### Copyright (C) 2024 Advanced Micro Devices, Inc.
```

Comment:
Please remove this line to avoid docs IO build issue.

Reply:
Deleted this line from the added files